html/includes/common/worldmap.inc.php

```diff
@@ -93,6 +95,30 @@
             </select>
         </div>
     </div>
+    <div class="form-group">
+        <div class="col-sm-4">
+            <label for="maintenance" class="control-label">Down devices over maintenance should be shown as: </label>
```

Comment:
Maybe `Down devices in maintenance windows shown as:`

Reply:
@laf: I removed option in 288edca